pkg/k2/constructs/template/construct_template_test.go

```diff
@@ -65,7 +67,7 @@ func TestParseConstructTemplateId(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			ctId, err := ParseConstructTemplateId(tt.input)
+			ctId, err := property.ParseConstructType(tt.input)
```

Comment:
Feels like this should be moved into the `property` package since it's testing a function from there. Same with a couple other tests here